File: Week1/hello /mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height : ");
    }
    while (height <= 0 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        for (int l = 0; l < 2; l++)
        {
            printf(" ");
        }
        for (int m = 0; m <= i; m++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week6/Problemset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height : ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    for j in range(height - 1, i, -1):
        print(" ", end = "")
    for k in range(0 , i + 1, 1):
        print("#", end = "")
    for l in range(2):
        print(" ", end = "")
    for m in range(0 , i + 1, 1):
        print("#", end = "")
    print("")